src/Microsoft.SqlTools.ServiceLayer/AzureFunctions/InsertSqlBindingOperation.cs

```diff
@@ -0,0 +1,143 @@
+
+                if (azureFunctionMethods.Count() == 0)
+                {
+                    return new ResultStatus()
+                    {
+                        Success = false,
+                        ErrorMessage = SR.CouldntFindAzureFunction(Parameters.functionName, Parameters.filePath)
+                    };
+                }
+                else if (azureFunctionMethods.Count() > 1)
+                {
+                    return new ResultStatus()
+                    {
+                        Success = false,
+                        ErrorMessage = SR.MoreThanOneAzureFunctionWithName(Parameters.functionName, Parameters.filePath)
+                    };
+                }
+
+                MethodDeclarationSyntax azureFunction = azureFunctionMethods.First();
+                var newParam = this.Parameters.bindingType == BindingType.input ? this.GenerateInputBinding() : this.GenerateOutputBinding();
+
+                // Generate updated method with the new parameter
+                // normalizewhitespace gets rid of any newline whitespace in the leading trivia, so we add that back
+                var updatedMethod = azureFunction.AddParameterListParameters(newParam).NormalizeWhitespace().WithLeadingTrivia(azureFunction.GetLeadingTrivia()).WithTrailingTrivia(azureFunction.GetTrailingTrivia());
+
+                // Replace the node in the tree
+                root = root.ReplaceNode(azureFunction, updatedMethod);
+
+                // write updated tree to file
+                var workspace = new AdhocWorkspace();
+                var syntaxTree = CSharpSyntaxTree.ParseText(root.ToString());
+                var formattedNode = CodeAnalysis.Formatting.Formatter.Format(syntaxTree.GetRoot(), workspace);
+                StringBuilder sb = new StringBuilder(formattedNode.ToString());
+                string content = sb.ToString();
+                File.WriteAllText(Parameters.filePath, content);
+
+                return new ResultStatus()
+                {
+                    Success = true
+                };
+            }
+            catch (Exception e)
+            {
+                return new ResultStatus()
+                {
+                    Success = false,
+                    ErrorMessage = e.ToString()
+                };
+            }
+        }
+
+        /// <summary>
+        /// Generates a parameter for the sql input binding that looks like
+        /// [Sql("select * from [dbo].[table1]", CommandType = System.Data.CommandType.Text, ConnectionStringSetting = "SqlConnectionString")] IEnumerable<Object> result
+        /// </summary>
+        private ParameterSyntax GenerateInputBinding()
+        {
+            // Create arguments for the Sql Input Binding attribute
+            var argumentList = SyntaxFactory.AttributeArgumentList();
+            argumentList = argumentList.AddArguments(SyntaxFactory.AttributeArgument(SyntaxFactory.IdentifierName($"\"select * from {Parameters.objectName}\"")));
```

Comment:
the object name should be quoted in [], or maybe there is already a util method for it.

Reply:
the object name in brackets will come from the sql database projects extension in vscode